### ISSUE TITLE
JAM Garage - Limbo Cars

### ISSUE THREAD

**Issue by lordmaca** [JustAnotherModder/JAM_Garage#9]
Hi Just,

All JOB cars in the server when players logout/crash or leave it (as well /dv) get to LIMBO.
I'm using JAM and JAM Garage, works all fine except for this matter.
Any Idea?

Looking in the code: JAG:ChangeState I saw the update to owned_vehicles at column jamstate=@state is that correct? (I changed to state=@state for testing but had no changes, I changed it back).

Also when the behavior above happens, at table owned_vehicles the state column goes to value 0 and that let the cars in the limbo ( I think).

I want to send all cars back to their garages when events above happen.

Thanks,
VP

**Comment by lordmaca** [comment]
Reason:

function JAG:GetPlayerVehicles(identifier)	
	local playerVehicles = {}
	local data = MySQL.Sync.fetchAll("SELECT * FROM owned_vehicles WHERE owner=@identifier",{['@identifier'] = identifier})	
	for key,val in pairs(data) do

		if not val.job or val.job == nil then
			local playerVehicle = json.decode(val.vehicle)
			table.insert(playerVehicles, {owner = val.owner, veh = val.vehicle, vehicle = playerVehicle, plate = val.plate, state = val.jamstate})
		end

	end
	return playerVehicles
end

**Comment by hyp3rstrike** [comment]
Yeah, this happens to me too. One of my players gave me a snippet of code to add to personal garages, but this doesn't affect the job garages.

**Comment by zelitao** [comment]
did you manage to solve ?? I have the same problem. the script is great but it gets useless like that.

**Comment by hyp3rstrike** [comment]
> did you manage to solve ?? I have the same problem. the script is great but it gets useless like that.

I solved it by changing garage scripts. Ended up changing to esx_advancedgarage before I stopped hosting FiveM RP servers.

**Comment by zelitao** [comment]
I managed to fix with the tip of the friend above, changing the job line. But still duplicating vehicles on impound, the ESX base really lacks a garage script.
